test/e2e/auth/service_accounts.go

```diff
@@ -485,7 +485,7 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 	   Containers MUST verify that the projected service account token can be
 	   read and has correct file mode set including ownership and permission.
 	*/
-	ginkgo.It("should set ownership and permission when RunAsUser or FsGroup is present [LinuxOnly] [NodeFeature:FSGroup]", func() {
+	ginkgo.It("should set ownership and permission when RunAsUser or FsGroup is present [LinuxOnly] [NodeFeature] [Feature:FSGroup]", func() {
```

Comment:
we don't need `NodeFeature`, do we? `Feature` + `sig-node` should be sufficient, right? issue mentions that we need to highlight "alpha" features, but it is separate.

Reply:
discussed offline. Need to come up with the plan how to mark these tests as this is not `sig-node` tests. Not clear what the `NodeFeature` tag will signify. It doesn't require any special test config or anything, I think

Reply:
/hold

adding feature tags means these tests will no longer run in default e2e runs, which is unexpected

This test is currently running in presubmit:
https://testgrid.k8s.io/google-gce#pull-kubernetes-e2e-gce-ubuntu-containerd&width=20&include-filter-by-regex=RunAsUser%20or%20FsGroup

But adding feature tags meant it got skipped in presubmit for this PR: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/103674/pull-kubernetes-e2e-gce-ubuntu-containerd/1418613003800023040/